### DISCUSSION TITLE
kangaroo/managed/harbor

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2067] [Announcements]
# kangaroo/managed/harbor

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kangaroo/managed/harbor.html

**Comment by liyuerich** [comment]
前提条件：需要先安装好 Cert Manager 

这里跳转到cert manager 首页，指导意义不大。 期望有一个安装的指导

**Reply by lemontrees123** [reply]
找到cert manager 一键安装即可，后续将在文档中补充说明

**Comment by liyuerich** [comment]
没有镜像存储的介绍？

**Reply by lemontrees123** [reply]
最佳实践中有镜像存储的资源规划：https://docs.daocloud.io/kangaroo/best-practice/capacity-planning.html

**Comment by windsonsea** [comment]
@lengrongfu @lemontrees123

**Comment by san3Xian** [comment]
database "core" 需要自行创建，文档和FAQ中好像没有提及
